content/pages/auth/login.md

```diff
@@ -1,9 +1,11 @@
 ---
 title: Welcome back
 description: Login to access all the features on TrollTracker
+loginLabel: Login
 signupPrompt: New to Trolltracker?
-signUpText: Sign up now
-signUpLink: /signup
+signupLink: /signup
+signupText: Sign up now
```

Comment:
```suggestion
signUpLink: /signup
signUpLabel: Sign up now
```
Or why are we calling this one `Text`?

Reply:
@kilemensi That was because the prop is not within the ` Button` component, but within a `Typography` component

Reply:
@kilemensi I'll update them to use a `...label` instead

Reply:
Oh. I think what we use to display the content, shouldn't matter it @nerdlet ... Button, Link, Typography, etc. That is, as long as it's not a long form text i.e. multiple lines/paragraph.